tests/storage/datavolume.go

```diff
@@ -356,8 +367,6 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 			})
 
 			It("[test_id:4643]should NOT be rejected when VM template lists a DataVolume, but VM lists PVC VolumeSource", func() {
```

Comment:
After this change this test doesn't test anything special and we might as well delete it.

Reply:
Hey Maya, 
I didn't change any logic in this test, only moved the creation of the vmi, dv to `beforeEach` scope and used libvmi instead of `NewRandomVMI` as we try to get rid of this function.

Reply:
I see